app/src/main/java/org/apache/fineract/data/models/loan/LoanAccount.kt

```diff
@@ -19,7 +19,8 @@ data class LoanAccount(
     @SerializedName("createdOn") var createdOn: String? = null,
     @SerializedName("createdBy") var createdBy: String? = null,
     @SerializedName("lastModifiedOn") var lastModifiedOn: String? = null,
-    @SerializedName("lastModifiedBy") var lastModifiedBy: String? = null
+    @SerializedName("lastModifiedBy") var lastModifiedBy: String? = null,
+    @SerializedName("isFakeData") var isFakeData: Boolean? = false
```

Comment:
I think it is not a good idea to add attributes to a retrofit model from the client-side and actually we can't modify it. if it is really necessary then it should be added by backend guys first.